site/content/installation/running-on-kind.md

```diff
@@ -105,4 +105,4 @@ kubectl -n nginx-gateway port-forward <pod-name> 8080:80 8443:443
 
 ## Get Started with NGINX Gateway Fabric
 
-Learn how to use NGINX Gateway Fabric by exploring the tutorials in the [examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.3.0/examples) directory. The guides provide practical instructions and scenarios to help you use NGINX Gateway Fabric effectively.
+Learn how to use NGINX Gateway Fabric by exploring the tutorials in the [examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.4.0/examples) directory. The guides provide practical instructions and scenarios to help you use NGINX Gateway Fabric effectively.
```

Comment:
Let's also update the version tag in this example manifest: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/installation/running-on-kind.md#set-up-a-nodeport

Reply:
Does the release workflow not catch manifest yaml in a document? Is this something that should be added to the workflow?

Reply:
As of right now, no. I think because it's not looking for the string "edge", because we include that in many places that we don't want updated.